ITEM.tsv

```diff
@@ -6968,56 +6968,56 @@ ITEM_20150918_006967	A crystal ball found in a wooden box at the monastery. Used
 ITEM_20150918_006968	A strong potion used for removing odors. It has an antiseptic scent to it.
 ITEM_20150918_006969	Scriptures that were scattered by rampaging monsters. Bring them to Mohaim.
 ITEM_20150918_006970	A torch received from Higgith. It is used to cremate bodies.
-ITEM_20151001_006971	
-ITEM_20151001_006972	
-ITEM_20151001_006973	
-ITEM_20151001_006974	
-ITEM_20151001_006975	
-ITEM_20151001_006976	
-ITEM_20151001_006977	
-ITEM_20151001_006978	
-ITEM_20151001_006979	
-ITEM_20151001_006980	
-ITEM_20151001_006981	
-ITEM_20151001_006982	
-ITEM_20151001_006983	
-ITEM_20151001_006984	
-ITEM_20151001_006985	
-ITEM_20151001_006986	
-ITEM_20151001_006987	
-ITEM_20151001_006988	
-ITEM_20151001_006989	
-ITEM_20151001_006990	
-ITEM_20151001_006991	
-ITEM_20151001_006992	
-ITEM_20151001_006993	
-ITEM_20151001_006994	
-ITEM_20151001_006995	
-ITEM_20151001_006996	
-ITEM_20151001_006997	
-ITEM_20151001_006998	
-ITEM_20151001_006999	
-ITEM_20151001_007000	
-ITEM_20151001_007001	
-ITEM_20151001_007002	
-ITEM_20151001_007003	
-ITEM_20151001_007004	
-ITEM_20151001_007005	
-ITEM_20151001_007006	
-ITEM_20151001_007007	
-ITEM_20151001_007008	
-ITEM_20151001_007009	
-ITEM_20151001_007010	
-ITEM_20151001_007011	
-ITEM_20151001_007012	
-ITEM_20151001_007013	
-ITEM_20151001_007014	
-ITEM_20151001_007015	
-ITEM_20151001_007016	
-ITEM_20151001_007017	
-ITEM_20151001_007018	
-ITEM_20151001_007019	
-ITEM_20151001_007020	
+ITEM_20151001_006971	- 8% chance to reduce damage by 50% when hit
+ITEM_20151001_006972	Shaman Doll Core
+ITEM_20151001_006973	A core that was used as a power source. Improper handling may result in a massive explosion. Acquired from a Shaman Doll.
+ITEM_20151001_006974	A core that was used as a power source. Improper handling may result in a massive explosion. Acquired from a Black Shaman Doll.
+ITEM_20151001_006975	Shaman Doll Cotton
+ITEM_20151001_006976	A cotton good for heat insulation. Acquired from a Shaman Doll.
+ITEM_20151001_006977	Comprehensive Research Data
+ITEM_20151001_006978	A lot of work was put into all of this research data.
+ITEM_20151001_006979	LV 3 Gem Abrasive
+ITEM_20151001_006980	LV 4 Gem Abrasive
+ITEM_20151001_006981	LV 5 Gem Abrasive
+ITEM_20151001_006982	Loxona Civil War Chapter 1
+ITEM_20151001_006983	The first book written about the Loxona Civil War. It appears to be legible.
+ITEM_20151001_006984	Loxona Civil War Chapter 2
+ITEM_20151001_006985	The second book written about the Loxona Civil War. It appears to be legible.
+ITEM_20151001_006986	A record of the Ruklys Era. It appears to be barely legible.
```

Comment:
"It appears to be barely legible." could also be changed to "It appears barely readable", it's shorter and it's consistent to the other descriptions since this text appears on other books as well.